Tangem/Common/Rounding/AmountRoundingType.swift

```diff
@@ -10,5 +10,9 @@ import Foundation
 
 enum AmountRoundingType {
     case shortestFraction(roundingMode: NSDecimalNumber.RoundingMode)
-    case `default`(roundingMode: NSDecimalNumber.RoundingMode)
+    case `default`(roundingMode: NSDecimalNumber.RoundingMode, scale: Int)
+
+    static func `default`(roundingMode: NSDecimalNumber.RoundingMode) -> AmountRoundingType {
+        return .default(roundingMode: roundingMode, scale: 2)
+    }
```

Comment:
а вроде можно 
```suggestion
    case `default`(roundingMode: NSDecimalNumber.RoundingMode, scale: Int = 2)
```
добавить дефолт параметр прям в enum